dbs_share_count.php

```diff
@@ -42,7 +42,7 @@ class DBSShareCount {
             $media_url;
 
     private $defaults = array (
-        "share_url" => WP_SITEURL . $_SERVER['REQUEST_URI'],
```

Comment:
@tjwallis We DO need to define the Request URI somewhere and add it here by default. Maybe define that variable above the $defaults array?